libs/sdk-ui-dashboard/src/model/commandHandlers/render/renderingWorker.ts

```diff
@@ -0,0 +1,178 @@
+// (C) 2021 GoodData Corporation
+import { Task, SagaIterator } from "redux-saga";
+import { put, delay, take, join, race, call, all, spawn, cancel, actionChannel } from "redux-saga/effects";
+import { DashboardContext } from "../../types/commonTypes";
+import { newDashboardEventPredicate, renderResolved, renderRequested } from "../../events";
+
+function* wait(ms: number): SagaIterator<true> {
+    yield delay(ms);
+    return true;
+}
+
+const isAsyncRenderResolvedEvent = (id?: string) =>
+    newDashboardEventPredicate(
+        "GDC.DASH/EVT.RENDER.ASYNC.RESOLVED",
+        id ? (e) => e.payload.id === id : () => true,
+    );
+
+const isAsyncRenderRequestedEvent = (id?: string) =>
+    newDashboardEventPredicate(
+        "GDC.DASH/EVT.RENDER.ASYNC.REQUESTED",
+        id ? (e) => e.payload.id === id : () => true,
+    );
+
+/**
+ * @internal
+ */
+export interface RenderingWorkerConfiguration {
+    /**
+     * Maximum time limit for rendering the dashboard.
+     *
+     * Default: 60000 (6s).
```

Comment:
This comment is probably wrong either 60 seconds or 6_000 as the default.

Reply:
Good catch, I'll correct it.